tasks/promote.yml

```diff
@@ -11,16 +11,16 @@
     cacert: "{{ os_images_cacert | default(omit) }}"
     interface: "{{ os_images_interface | default(omit, true) }}"
   register: retire_list
-  when: "item.rename_image | default(os_images_promote) | bool"
+  when: item.rename_image | default(os_images_promote) | bool
 
 - name: Ensure images for retirement exist
   assert:
     that:
       - item.name in retire_list.openstack_images | map(attribute='name') | list
-    fail_msg: "'The image {{ item.name[:-(os_images_name_suffix | length | int)] }} does not exist."
+    fail_msg: "The image {{ item.name[:-(os_images_name_suffix | length | int)] }} does not exist."
     success_msg: "The image '{{ item.name[:-(os_images_name_suffix | length | int)] }}' exists in the cloud."
```

Comment:
Do we need it? We just need to fail on the images that do not exist (with a message), sucess_msg is not needed.